src/algebra/big_operators/order.lean

```diff
@@ -308,3 +309,70 @@ open opposite
 (@unop_add_hom β _).map_sum _ _
 
 end with_top
+
+namespace finset
+
```

Comment:
Is there a reason these are in the `algebra/big_operators/order` file? Their presence should be clarified in the module doc if there's a reason -- I would never expect to find them here. But maybe they belong somewhere else.

Reply:
They're only here for my own convenience: the proof uses big operators and `sum_le_sum`.  It's seemed to me they ought to be elsewhere, but I'm not sure where that might be!  Maybe create `combinatorics/basic`?